manifests/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```

Comment:
nit: The service account should be above / in the deployment file. Otherwise there is a race condition on the pods being created and the service account being created which *will* cause failure events for a short period after deployment.